### ISSUE TITLE
Safe Annotations: constrained_annotations is array instead of object

### ISSUE THREAD

**Issue by kravciak** [kubewarden/ui#307]
### Current Behavior

Policy default yaml contains `constrained_annotations: []`. Creating policy with this settings shows error:

```
settings are not valid: Some("Provided settings are not valid: parse error: expected { near offset 28 of 'constraine...'")
```

If I touch settings value is changed to object `constrained_annotations: {}`. This is accepted and policy starts fine.

Note from @Martin-Weiss - same issue happens on safe-labels policy

[Screencast from 2023-03-27 19-42-29.webm](https://user-images.githubusercontent.com/538604/228023948-a148ec17-a352-4f66-a430-b9530fb80e30.webm)

**Comment by Martin-Weiss** [comment]
Same issue exists in safe-labels policy as well..

**Comment by kravciak** [comment]
Fixed in `safe-annotations:v0.2.7` & `safe-labels:v0.1.12`
